src/pages/admin/programs/components/program-section-form/ProgramSectionForm.test.tsx

```
@@ -120,7 +144,46 @@ describe('ProgramSectionForm', () => {
+                            (c: any) =>
+                                allowed.has(c.contentType) &&
+                                c.groupIndex !== null &&
+                                c.groupIndex !== undefined &&
+                                typeof c.groupIndex === 'number',
+                        )
+                        .map((c: any) => c.groupIndex as number),
+                ),
+            ).sort((a, b) => a - b);
+
+            const map = new Map<number, number>();
+            groups.forEach((g, i) => map.set(g, i));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)